mixer/pkg/api/grpcServer_test.go

```diff
@@ -288,7 +347,13 @@ func TestReport(t *testing.T) {
 		return nil
 	}
 
-	request := mixerpb.ReportRequest{Attributes: []mixerpb.CompressedAttributes{{}}}
+	request := mixerpb.ReportRequest{Attributes: []mixerpb.CompressedAttributes{}}
+	_, err = ts.client.Report(context.Background(), &request)
+	if err != nil {
+		t.Errorf("Expected success, got error: %v", err)
+	}
+
+	request = mixerpb.ReportRequest{Attributes: []mixerpb.CompressedAttributes{{}}}
```

Comment:
Seems like line 350 & 356 are same?

Reply:
Subtelly different, the second case has an extra set of braces...